coral-component-actionbar/src/scripts/ActionBar.js

```diff
@@ -556,7 +556,14 @@ class ActionBar extends BaseComponent(HTMLElement) {
   
     const primary = this._elements.primary;
     const secondary = this._elements.secondary;
-  
+
+    if(!primary.getAttribute('role')) {
```

Comment:
Can you please add space after `if` statement ?

Reply:
Done.